profiles/SampleBarbarian-setup.yaml

```diff
@@ -1,131 +1,604 @@
----
-# https://elanthipedia.play.net/Barbarian#Circle_Requirements
-# 5 Weapons (only 4 are needed to circle): Brawling, Heavy Thrown, Polearms, Large Edged, Twohanded Edged
-# 2 Armors: Light Armor, Shield Usage
-# 4 Survivals: Athletics, Perception, Skinning, Stealth (Does not include Evasion)
-# 1 Lore: Appraisal
-# 1 Supernatural: Augmentation
-
-# Eventually you will need to train 1 more Supernatural to circle (for a total of 2)
-# However, you will probably end up training them all since they each have very useful abilities
-# The other Supernatural skills are: Warding, Debilitation
-
-# Hunting settings
-training_manager_hunting_priority: true
+# NOTES:
+#    - You can find all the BASE settings inside the base.yaml within this same folder! Search the file for keywords for more settings!
+#    - If you can "TAP ADJECTIVE NOUN" for an item, then you can use that for the adjective & noun of the item within the scripts!
+
+# High-level example YAMLS:
+#  - Chuno
+
+# This file can be used with ;training-manager or ;t2.  The T2 specifics are at the very bottom.
+
+hometown: Crossing
+
+#all the settings to make you do allll the stuff in your safe room versus random rooms around town
+safe_room: 8246
```

Comment:
❓ What room is `8246` and are we ok with a bunch of newbies congregating there?

Reply:
It's just a random room in the empath garden.  Shouldn't matter but it could be made blank and commented out

Reply:
We have a lot of these types of things in the base settings, outside WM's, Ranger's guild, etc.

Reply:
At least they can be kicked from the Empath guild :)

Reply:
I'm fine with a reasonable default room. People will figure out they need to change it quickly enough (or not and face the consequences).